common/queue/src/main/java/org/thingsboard/server/queue/usagestats/DefaultTbApiUsageReportClient.java

```diff
@@ -110,31 +120,50 @@ private void reportStats() {
                 });
 
```

Comment:
The good point done is to replace string keys with proto.

I would like to see the resulted message with Kafka UI. To be sure that the output message has no duplicated data.
Unfortunately, I see the duplicated data . Please, visualise the output to proof yourself that the optimisation job done.

here are two hints on the picture. (the service id, tenantId if the customer stats enabled) 

![image](https://github.com/user-attachments/assets/f70e609a-d6f9-474e-8c35-10d9d7a529c3)

Reply:
I agree with serviceId, but with tenantId no, because if we create some group msg per one tenant, this message will include stats from all customers per tenant so this message can be huge and will processed in one tb-core.